oat-sa/tao-test-runner-qti-fe
# Feature/actp 303/review panel fix html structure

**Jira ticket**
https://oat-sa.atlassian.net/browse/ACTP-303

**Description**
The review panel consists of a multitude of different regions/landmarks and is nested in another one. Screen readers only announce the lowest Landmark/region level, so the info about the higher levels is lost.

**How to test**
1. Create a test
2. Start a test as a testtacker
3. Enable screenreader
4. When the focus arrives to the review panel the header should be readable

**Companion PR**
https://github.com/oat-sa/extension-tao-act/pull/1337
